gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -236,27 +244,84 @@ def call_load(self, other_args):
             required="-h" not in other_args,
         )
         parser.add_argument(
-            "-s",
             "--source",
             help="Source of data",
             dest="source",
             choices=("cp", "cg", "bin", "cb"),
             default="cg",
             required=False,
         )
+        parser.add_argument(
+            "-s",
+            "--start",
+            type=valid_date_in_past,
+            default=(datetime.now() - timedelta(days=366)).strftime("%Y-%m-%d"),
+            dest="start",
+            help="The starting date (format YYYY-MM-DD) of the crypto",
+        )
+
```

Comment:
not important but in future would be nice to not have gaps between the `add_argument` cases